themes/bootstrap3/templates/search/facet-list-content.phtml

```diff
@@ -0,0 +1,21 @@
+<div class="full-facet-list facet-group<?php if(!$active): ?> hidden<?php endif; ?>" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
+  <?php if ($active): ?>
+    <?php if ($this->page > 1 && !$this->inLightbox && !$this->ajaxLightbox): ?>
+      <a href="<?=$urlBase . '&amp;facetpage=' . ($this->page - 1) . '&amp;facetsort=' . urlencode($this->sort) . '&amp;contains=' . urlencode($this->contains)?>" class="facet js-facet-prev-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->sort ?>" data-limit="<?=count($this->data) ?>"><?=$this->translate('prev_ellipsis') ?></a>
```

Comment:
The URLs in this template probably ought to be reformatted to match the URLs in facet-list.phtml for consistency (i.e. remove hard-coded escaped ampersands and wrap in escapeHtmlAttr).